component/core/src/main/java/org/exoplatform/social/core/jpa/storage/dao/jpa/SpaceDAO.java

```diff
@@ -394,6 +399,12 @@ private void buildPredicates(XSpaceFilter spaceFilter,
       parameterNames.add(PARAM_USER_ID);
     }
 
+    if (spaceFilter.getParentSpaceId() > 0) {
+      suffixes.add("ParentSpaceId");
+      predicates.add("s.parentSpaceEntity.id = :parentSpaceId");
```

Comment:
After deleting the entity mapping as suggested above
 
```suggestion
      predicates.add("s.parentSpaceId = :parentSpaceId");
```